include/jemalloc/internal/peak.h

```diff
@@ -0,0 +1,32 @@
+#ifndef JEMALLOC_INTERNAL_PEAK_H
+#define JEMALLOC_INTERNAL_PEAK_H
+
+typedef struct peak_s peak_t;
+struct peak_s {
+	uint64_t cur_max;
+	uint64_t adjustment;
+};
```

Comment:
Could you add some comments on the meaning of the two fields?

Reply:
Done.